.travis.yml

```diff
@@ -0,0 +1,30 @@
+sudo: required
+dist: trusty
+language: node_js
+env:
+  global:
+  # - FF_BETA_PATH: './firefox/firefox'
+node_js:
+  - 'stable'
+addons:
+  # From some tests it seems you can only have one firefox addon.
+  # Adding two results in the last one defined being the one installed.
+  firefox: latest
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+      - google-chrome-beta
+
+before_install:
+  # Get firefbox beta and extract to tar for the test runner to use.
+  # - wget 'https://download.mozilla.org/?product=firefox-beta-latest&lang=en-US&os=linux64' -O firefox-beta.tar.bz2 && tar xvjf firefox-beta.tar.bz2
+
+# Read more here: https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+
+script:
+  - gulp build && gulp lint test
```

Comment:
I want the `build` to take place first, and if I list them all together then they'll be run concurrently. If there's a better way to get the `build` to run first (maybe via one of the other lifecycle events in Travis?) I'm happy to switch that out, but this seemed to work.